apps/web/src/components/Layout/Layout.tsx

```diff
@@ -88,6 +89,22 @@ const Layout = () => {
         })}
       </MenuLink>,
     ];
+    if (
+      loggedInUser?.legacyRoles &&
+      loggedInUser.legacyRoles.length > 0 &&
+      loggedInUser.legacyRoles.includes(LegacyRole.Admin)
+    ) {
+      const adminLink = (
+        <MenuLink key="adminDashboard" to={paths.adminDashboard()}>
+          {intl.formatMessage({
+            defaultMessage: "Admin",
+            id: "wHX/8C",
+            description: "Title tag for Admin site",
+          })}
+        </MenuLink>
+      );
+      menuItems.push(adminLink);
```

Comment:
Usually we prefer immutable operations.  Note how on line 74 a new array is created by spreading the previous items and aadding new items rather than modifying the existing array.

Reply:
okay, done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/5703/commits/05d273e30c6c2e2fd72b2b85f70dde059a72ad01